### PR TITLE
update code to support rosa upgrade for 4.16

### PR DESCRIPTION
**Pull request by liqcui**
update code to support rosa upgrade for 4.16

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15-loaded-upgrade416-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15-loaded-upgrade416-24nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-candidate-x86-loaded-upgrade-from-4.15-loaded-upgrade416-24nodes

**Comment by svetsa-rh** [comment]
@afcollins Can you pls help suggest crons for these jobs. They should be bi-weekly. Thanks!

**Comment by svetsa-rh** [comment]
@liqcui I see all crons scheduled at same time - same days of the month and same hour as well. We are better off spreading them out. I have requested Andrew to suggest. It seems like you are partially following the doc already. 

lgtm otherwise, thanks for adding these.

**Comment by liqcui** [comment]
@afcollins @svetsa-rh

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      channel: candidate
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade416-24nodes
+  cron: 0 2 3,17 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml:34]: suggestion: `"0 18 3,17 * *"`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade415-24nodes
+  cron: 0 1 10,24 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml:34]: suggest: `"0 20 10,24 * *"`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      channel: candidate
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade416-24nodes
+  cron: 0 6 10,24 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml:34]: suggest: `"0 16 10,24 * *"`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade415-24nodes
+  cron: 0 10 3,17 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml:34]: suggest: `"0 14 3,17 * *"`

**Review by afcollins — COMMENTED** [review]
Looks good! I think the days are fine. My suggestions are only to spread the hours out a little more over the day.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml (hunk 1 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml]: updated in new code, thanks!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml (hunk 2 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml]: updated in new code, thanks!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml (hunk 3 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.16-candidate-x86-loaded-upgrade-from-4.15.yaml]: updated in new code, thanks!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml (hunk 4 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-candidate-x86-loaded-upgrade-from-4.14.yaml]: updated in new code, thanks!

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.14-stable-x86-loaded-upgrade-from-4.14-loaded-upgrade414-24nodes

**Review by afcollins — APPROVED** [review]
/lgtm

**Pull request merged**